Update Tensorflow filter plugin doc

Update needed for Tensorflow filter plugin doc based on code here https://github.com/fluent/fluent-bit/blob/master/plugins/filter_tensorflow/tensorflow.c.

@cosmo0920 I thought I'd ask you in this as I was turning on the build of the Tensorflow filter plugin in the source build by adding:

`-DFBL_FILTER_TENSORFLOW='On'`

But it's failing with missing libs?

`
[ 65%] Building C object plugins/filter_tensorflow/CMakeFiles/flb-plugin-filter_tensorflow.dir/tensorflow.c.o
/Users/erics/demo-projects/workshops/fluentbit-install-demo/target/fluent-bit-4.1.0/plugins/filter_tensorflow/tensorflow.c:30:10: fatal error: 'tensorflow/lite/c/c_api.h' file not found
   30 | #include "tensorflow/lite/c/c_api.h"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
make[2]: *** [plugins/filter_tensorflow/CMakeFiles/flb-plugin-filter_tensorflow.dir/tensorflow.c.o] Error 1
make[1]: *** [plugins/filter_tensorflow/CMakeFiles/flb-plugin-filter_tensorflow.dir/all] Error 2
make: *** [all] Error 2
[ERROR]    Error occurred with 'make' command...
`

Is this just broken or am I doing something wrong?

--- a/pipeline/filters/tensorflow.md
+++ b/pipeline/filters/tensorflow.md
@@ -13,12 +13,12 @@ The Tensorflow plugin for Fluent Bit has the following limitations:
 
 The plugin supports the following configuration parameters:
 
-| Key | Description | Default |
-| :--- | :--- | :--- |
-| `input_field` | Specify the name of the field in the record to apply inference on. | _none_ |
-| `model_file` | Path to the model file (`.tflite`) to be loaded by Tensorflow Lite. | _none_ |
-| `include_input_fields` | Include all input filed in filter's output. | `True` |
-| `normalization_value` | Divide input values to `normalization_value`. | _none_ |
+| Key                    | Description                                                         | Default |
+|:-----------------------|:--------------------------------------------------------------------|:--------|
+| `include_input_fields` | Include all input filed in filter's output.                         | `True`  |
+| `input_field`          | Specify the name of the field in the record to apply inference on.  | _none_  |
+| `model_file`           | Path to the model file (`.tflite`) to be loaded by Tensorflow Lite. | _none_  |
+| `normalization_value`  | Divide input values to `normalization_value`.                       | _none_  |
 
 ## Creating a Tensorflow Lite shared library
 
@@ -32,8 +32,8 @@ To create a Tensorflow Lite shared library:
    bazel build -c opt //tensorflow/lite/c:tensorflowlite_c  # see https://github.com/tensorflow/tensorflow/tree/master/tensorflow/lite/c
    ```
 
-   The script creates the shared library
-   `bazel-bin/tensorflow/lite/c/libtensorflowlite_c.so`.
+   The script creates the shared library `bazel-bin/tensorflow/lite/c/libtensorflowlite_c.so`.
+  
 1. Copy the library to a location such as `/usr/lib` that can be used by Fluent Bit.
 
 ## Building Fluent Bit with Tensorflow filter plugin

